examples/acquiring_data/downloading_lascoC2.py

```diff
@@ -1,41 +0,0 @@
-"""
-======================================
-Downloading and plotting LASCO C2 data
```

Comment:
I get that this isn't doing anything particularly "interesting" but if someone wants to get a C2 image they are going to find this and follow it. What's the harm in keeping it?